Sources/SwiftInspectorVisitors/PropertyInfo.swift

```diff
@@ -1,4 +1,90 @@
 // Created by Michael Bachand on 9/15/21.
 // Copyright © 2021 Airbnb Inc. All rights reserved.
 
-import Foundation
+// MARK: - PropertyInfo
+
+public struct PropertyInfo: Codable, Hashable, CustomDebugStringConvertible {
```

Comment:
This type is big enough for its own file, especially considering that we are adding a custom `Codable` conformance that we will test.